src/libraries/System.Memory/src/System/Buffers/Text/Base64Decoder.cs

```diff
@@ -5,12 +5,13 @@
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.Arm;
 using System.Runtime.Intrinsics.X86;
 
 namespace System.Buffers.Text
 {
     // AVX2 version based on https://github.com/aklomp/base64/tree/e516d769a2a432c08404f1981e73b431566057be/lib/arch/avx2
-    // SSSE3 version based on https://github.com/aklomp/base64/tree/e516d769a2a432c08404f1981e73b431566057be/lib/arch/ssse3
+    // Vector128 version based on https://github.com/aklomp/base64/tree/e516d769a2a432c08404f1981e73b431566057be/lib/arch/ssse3
```

Comment:
I am wondering did you try the latest implementation at https://github.com/aklomp/base64/tree/master/lib/arch/ssse3?

Reply:
Hmm... I hadn't noticed there was a new SSE3 version there. A very quick glance the code does look better in the new version.

Reply:
>https://github.com/aklomp/base64/tree/master/lib/arch/ssse3? Is it any better?

This latest version is the exact same algorithm. The C++ code has just been moved around, but the outputted assembly is the same. It's the same with the Encode routine too.

There is a difference in the way the code is called. It uses rounds to optimise the number of calls to the inner loop:
https://github.com/aklomp/base64/blob/bce78a981ffb5a0973c513f41761e6c6a77d8f5d/lib/arch/ssse3/dec_loop.c#L120
(and the equivalent in encode too)

It's probably worth an investigation to see if it's worthwhile doing here. Given it doesn't really intersect with the code in this PR, then I'd highly recommend that move to a different task/PR.  (Also, I don't know if that was originally investigated for the original dotnet SSE3 version)

Reply:
sounds good to me.